src/main/java/net/neoforged/neoforge/registries/ModifyRegistryEvent.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.registries;
+
+import net.minecraft.core.Registry;
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.resources.ResourceKey;
+import net.neoforged.bus.api.Event;
+import net.neoforged.fml.event.IModBusEvent;
+import org.jetbrains.annotations.ApiStatus;
+
+/**
+ * Fired for every builtin registry and datapack registry after they are constructed.
+ * For builtin registries, this event is fired after vanilla entries are registered but before modded entries.
+ * For datapack registries, this event is fired before any entries are registered.
```

Comment:
This is completely wrong, the event does not fire at all for datapack registries. That might have been the case already since 1.19.3. Should we just remove this javadoc?

Reply:
Went ahead and removed the javadoc - registry callbacks for datapack registries can be done in another PR due to requiring a number of new patches.